cdhweb/static_src/global/css-variables/layout.scss

```diff
@@ -10,6 +10,7 @@
   --standard-gap: 16px;
   --page-gutter: 16px;
   --content-max-width: #{px2rem(1770)};
+  --reading-max-width: #{px2rem(865)};
```

Comment:
I usually see this as something like `--max-line-length` — is this naming so that it's linked/comparable to `--content-max-width`?

Reply:
Yeah, something like that